dotcom-rendering/src/web/components/ElementContainer.tsx

```diff
@@ -79,7 +87,9 @@ export const ElementContainer = ({
 				</div>
 			);
 			const style = css`
-				${backgroundColour && setBackgroundColour(backgroundColour)}
+				${backgroundColour && setBackgroundColour(backgroundColour)};
+				${backgroundColourMobile &&
+				setBackgroundColourMobile(backgroundColourMobile)};
```

Comment:
What happens when both `backgroundColour` and `backgroundColourMobile` are passed?

Reply:
So if both are passed in you get a main style and a style wrapped in a media query, like this:
```
background-color: ${backgroundColour};
${until.desktop} {
	background-color: ${backgroundColourMobile};
}
```